lib/twilio-ruby/twiml/twiml.rb

```diff
@@ -26,13 +26,19 @@ class TwiML
         attr_accessor :name
 
         def initialize(**keyword_args)
+          @overrides = {
+              'aliasAttribute': 'alias',
+              'xmlLang': 'xml:lang',
+              'interpretAs': 'interpret-as',
+          }
           @name = self.class.name.split('::').last
           @value = nil
           @verbs = []
           @attrs = {}
 
           keyword_args.each do |key, val|
-            @attrs[TwiML.to_lower_camel_case(key)] = val unless val.nil?
+            key = @overrides.fetch(key, TwiML.to_lower_camel_case(key))
```

Comment:
I would probably rename the `key` you're assigning on this line to, like, `corrected_key` for readability, but I don't want to hold this up, so feel free to merge either way.